doc/devel/MEP/MEP27.rst

```diff
@@ -103,30 +105,30 @@ The description of this MEP gives us most of the solution:
 |FigureManagerBase(canvas, num)        |FigureManager(figure, num)    |``WindowBase(title)``|Notes                           |
 |                                      |                              |                     |                                |
 +======================================+==============================+=====================+================================+
-|show                                  |                              |show                 |                                |
+|show                                  |->                            |show                 |                                |
 +--------------------------------------+------------------------------+---------------------+--------------------------------+
 |destroy                               |calls destroy on all          |destroy              |                                |
 |                                      |components                    |                     |                                |
 +--------------------------------------+------------------------------+---------------------+--------------------------------+
 |full_screen_toggle                    |handles logic                 |set_fullscreen       |                                |
 +--------------------------------------+------------------------------+---------------------+--------------------------------+
-|resize                                |                              |resize               |                                |
+|resize                                |->                            |resize               |                                |
 +--------------------------------------+------------------------------+---------------------+--------------------------------+
-|key_press                             |key_press                     |                     |                                |
+|key_press                             |key_press                     |X                    |                                |
 +--------------------------------------+------------------------------+---------------------+--------------------------------+
-|show_popup                            |show_poup                     |                     |Not used anywhere in mpl, and   |
+|show_popup                            |show_poup                     |X                    |Not used anywhere in mpl, and   |
```

Comment:
I actually think I will get rid of this altogether now, a carry over from the ``FigureManagerBase``, which only returned ``pass``, presumably someone had plans for it but never realised them.  As we now rcParam this new class I think we can safely drop it.